packages/scroller/src/vaadin-scroller.js

```diff
@@ -110,6 +110,38 @@ class Scroller extends FocusMixin(ElementMixin(ControllerMixin(ThemableMixin(Pol
   _shouldSetFocus(event) {
     return event.target === this;
   }
+
+  /**
+   * Scrolls the textarea to the start if it has a vertical scrollbar.
+   */
+  scrollToStart() {
+    switch (this.scrollDirection) {
+      case 'vertical':
+        this.scrollTop = 0;
+        break;
+      case 'horizontal':
+        this.scrollLeft = 0;
+        break;
+      default:
+        break;
+    }
+  }
+
+  /**
+   * Scrolls the textarea to the end if it has a vertical or horizontal scrollbar.
+   */
+  scrollToEnd() {
+    switch (this.scrollDirection) {
+      case 'vertical':
+        this.scrollTop = this.scrollHeight;
+        break;
+      case 'horizontal':
+        this.scrollLeft = this.scrollWidth;
```

Comment:
For proper RTL support, please use `setNormalizedScrollLeft` helper. 

https://github.com/vaadin/web-components/blob/fab94595fd5e8c11c0709413251be92994a1daa9/packages/component-base/src/dir-utils.js#L30

Note: you can test the RTL mode in dev page by setting `dir="rtl"` attribute on the `<html>` element.